skillserver/skillserver.go

```diff
@@ -58,12 +59,26 @@ func SetRootPrefix(prefix string) {
 	RootPrefix = prefix
 }
 
-func RunSSL(apps map[string]interface{}, port, cert, key string) error {
-	router := mux.NewRouter()
-	Init(apps, router)
-
-	err := http.ListenAndServeTLS(port, cert, key, router)
-	return err
+func RunSSL(apps map[string]interface{}, port, cert, key string) {
+  router := mux.NewRouter()
+  Init(apps, router)
+
+  cfg := &tls.Config{
```

Comment:
Might be useful to doc what this is doing here (allowing self-signed certs? anything else?). I'm not sure I quite understand what's going on.